app/assets/javascripts/spree/backend/solidus_stripe.js

```diff
@@ -0,0 +1,2 @@
+// Placeholder manifest file.
```

Comment:
I think you can remove this and other empty asset files.

Reply:
sure. I was thinking that maybe those empty files may be handy in case somebody in the future needs to edit them. But surely they can add them back, why keep all that now-useless bolierplate around?